Modules.md

```diff
@@ -140,6 +140,14 @@ The linear memory section may also contain an optional hint declaring the expect
 maximum heap usage. This hint is not semantically visible but can help a
 WebAssembly engine to optimize `grow_memory`.
 
+The linear memory section may optionally declare that the instance's
+linear memory is *externally aliasable*. How linear memory is aliased is up
+to the host environment (as with all module exports). The 
+[Web](Web.md#aliasing-linear-memory-from-JS) would reflect exported linear
+memory to JS as an `ArrayBuffer`. The MVP does not currently provide for
+*importing* linear memory though this may be added 
+[in the future](FutureFeatures.md#importing-linear-memory).
```

Comment:
The text reads as "oh I can just say externally alias and then everything works" but I see it as an undefined hole in the memory model. A hole we can plug, but will require some pretty serious design work that I'm not sure we want to tackle for MVP considering C++ hasn't gotten it right either.

That being said, maybe we can go with it for now, I can open an issue, and we can revisit when we have a clearer idea backed by code?